resources/splunk_service.rb

```diff
@@ -0,0 +1,158 @@
+# Cookbook Name:: cerner_splunk_ingredient
+# Resource:: splunk_service
+#
+# Resource for managing Splunk as a system service
+class SplunkService < ChefCompat::Resource
+  include CernerSplunk::PlatformHelpers, CernerSplunk::ResourceHelpers,
+          CernerSplunk::ServiceHelpers, CernerSplunk::RestartHelpers
+
+  resource_name :splunk_service
+
+  property :name, String, name_property: true, identity: true
+  property :package, [:splunk, :universal_forwarder], required: true
+  property :options, String, default: ''
```

Comment:
I'm handling `--accept-license` already. I didn't think those flags were first run only, if that's the case there's no point having options there. I might add no-prompt to my first run commands, and hopefully any issues that involve prompting at first run can be resolved outside of that prompt.